### DISCUSSION TITLE
Developer Advocacy

### DISCUSSION THREAD

**Discussion by Akankshabhasin** [EddieHubCommunity/support#617] [General]
I was exploring Developer Advocacy these days, so wanted to know do companies also hire fresher for this role and what are the roles and responsibilities?
cc- @eddiejaoude

**Comment by eddiejaoude** [comment]
great question @Akankshabhasin  👍  - previously I felt companies did not, but more recently, I am seeing companies realise the value of different levels of experiences for DevRel / Dev Advocacy. So in short yes it is possible :nerd: . I think the people already doing it, will have a great chance of success. I know you have a youtube channel already, so this is great :muscle:
if I was hiring and I had 2 candidates 
- more senior but no youtube / blog / open source etc
- more junior but had youtube / blog / open source etc
I know I would hire the more junior person as the right attitude, communication, collaborating, community etc I believe to be harder to learn than tech

**Reply by Akankshabhasin** [reply]
Thank you so much @eddiejaoude 😄

**Reply by eddiejaoude** [reply]
No problem.

@Akankshabhasin I recommend adding a profile picture to your GitHub account to make it look more professional (I can't remember if I mentioned that already or not, sorry if I did)
